bears/rest/RSTcheckBear.py

```diff
@@ -32,16 +33,30 @@ class RSTcheckBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         code_block_language_ignore: list=()):
+                         code_block_language_ignore: list=(),
+                         directive_ignore: typed_list(str)=(),
+                         role_ignore: typed_list(str)=()):
         """
         :param code_block_language_ignore:
             Comma seperated list for languages of which code blocks
             should be ignored. Code block of following languages
             can be detected: ``bash``, ``c``, ``cpp``, ``json``,
             ``python``, ``rst``.
+        :param directive_ignore:
+             Comma separated list of directives to be ignored. Can be
+             used to ignore custom directives.
+        :param role_ignore:
+             Comma separated list of roles to be ignored. Can be used
+             to ignore custom roles.
         """
         args = ()
         if code_block_language_ignore:
             args = ('--ignore-language=' +
                     ','.join(code_block_language_ignore),)
+        if directive_ignore:
+            directives = ','.join(part.strip() for part in directive_ignore)
```

Comment:
like said, you don't need `part.strip()`, just do `','.join(directive_ignore)`.
You can even inline it into the next line:
```python
args += ('--ignore-directives=' + ','.join(directive_ignore),)
```

Reply:
Sure.